### ISSUE TITLE
Deleting a server leaves DRBD constraints in the CIB

### ISSUE THREAD

**Issue by digimer** [ClusterLabs/anvil#374]
```
<node_attributes>
  <node name="an-a01n02">
    <attribute name="drbd-fenced_an-test-deploy1" value="0"/>
    <attribute name="drbd-fenced_an-test-deploy2" value="0"/>
  </node>
</node_attributes>

```
This should be removed when a server is deleted

**Comment by digimer** [comment]
This is really odd... the instance attributes are removed from node 1, but not node 2... I wonder if this is a pcs bug? I can detect/remove though.

**Comment by digimer** [comment]
Should be resolved in 7258781.

**Comment by fabbione** [comment]
this fix is not correct in nature and it´s racy.

the delete-server should make sure that the attributes are removed from both nodes and not scan-server.

i delete a server, attributes are removed from nodeX and left on nodeY, I create a server with the same name before scan-server run, kaboom.

This is going to cause issues with starting servers on the correct node.

**Comment by digimer** [comment]
Ok, I put the fox in scan-cluster, but that's not fast enough. I'll move the fix into the delete tool.

**Comment by digimer** [comment]
This should now be fixed in commit 6ee2ad7

**Comment by fabbione** [comment]
this is fixed in pr #376
